extensions/provisioning/customcluster.go

```diff
@@ -12,3 +15,10 @@ type CustomClusterConfig struct {
 	SpecifyPublicIP      bool                 `json:"specifyPublicIP" yaml:"specifyPublicIP"`
 	NodeNamePrefix       string               `json:"nodeNamePrefix" yaml:"nodeNamePrefix"`
 }
+
+func GetInstallDockerURL() string {
```

Comment:
Write a comment here please of our go docs.

Reply:
Added the comment.